### PR TITLE
[nrf fromlist] [nrfconnect] Add kconfig for enabling Read Client

### PR DESCRIPTION
**Pull request by ArekBalysNordic**
To save some FLASH and RAM space we can disable Read Client functionality in nrfconnect samples.
This commit adds kconfig to enable or disable Read Client in the Interaction Model.

### CODE REVIEW

**Review by markaj-nordic — APPROVED** [review]
Does this affect dynamic endpoints with client clusters as well?

**Comment by ArekBalysNordic** [comment]
> Does this affect dynamic endpoints with client clusters as well?

In Matter Mridge we use bindings, so we need to enable it there anyway.

**Comment by markaj-nordic** [comment]
> > Does this affect dynamic endpoints with client clusters as well?
> 
> In Matter Bridge we use bindings, so we need to enable it there anyway.

Yes, this is what I wanted to confirm, if this config enabled read client functionality also on dynamic clusters. Thx,

**Review by LuDuda — APPROVED** [review]

File: examples/light-switch-app/nrfconnect/prj.conf [hunk 1]
```diff
@@ -45,3 +45,6 @@ CONFIG_USE_SEGGER_RTT=n
 # Enable Factory Data feature
 CONFIG_CHIP_FACTORY_DATA=y
 CONFIG_CHIP_FACTORY_DATA_BUILD=y
+
+# Enable the Read Client for binding Purposes
```
Comment (LuDuda) [inline examples/light-switch-app/nrfconnect/prj.conf:49]:
```suggestion
# Enable the Read Client for binding purposes
```

File: examples/light-switch-app/nrfconnect/prj_no_dfu.conf [hunk 2]
```diff
@@ -47,3 +47,6 @@ CONFIG_CHIP_OTA_REQUESTOR=n
 
 # Disable QSPI NOR
 CONFIG_CHIP_QSPI_NOR=n
+
+# Enable the Read Client for binding Purposes
```
Comment (LuDuda) [inline examples/light-switch-app/nrfconnect/prj_no_dfu.conf:51]:
```suggestion
# Enable the Read Client for binding purposes
```

File: examples/light-switch-app/nrfconnect/prj_release.conf [hunk 3]
```diff
@@ -59,3 +59,6 @@ CONFIG_BOOT_BANNER=n
 # Enable Factory Data feature
 CONFIG_CHIP_FACTORY_DATA=y
 CONFIG_CHIP_FACTORY_DATA_BUILD=y
+
+# Enable the Read Client for binding Purposes
```
Comment (LuDuda) [inline examples/light-switch-app/nrfconnect/prj_release.conf:63]:
```suggestion
# Enable the Read Client for binding purposes
```

**Review by kkasperczyk-no — APPROVED** [review]

**Pull request merged**